etc/org.opencastproject.adminui.endpoint.SeriesEndpoint.cfg

```diff
@@ -14,3 +14,13 @@
 # to in the events filter.
 # Default: true
 #eventsFilter.onlySeriesWithWriteAccess=true
+
+# The base URL of the Tobira instance you want to be able to publish series to.
+# You need to configure this for the Tobira panel to show up in the series creation wizard.
+#tobira.origin=http://localhost:3080
+
+# A shared secret between Tobira and Opencast to get authenticated access to the
+# `mount`-API. See also the `auth.trusted_external_key` configuration option in your
+# Tobira configuration.
+# You need to configure this for the Tobira panel to show up in the series creation wizard.
```

Comment:
Maybe add something like:

> Be aware that configuring this means that anyone with write access to the Tobira editors in the admin interface will have access to do modifications to any part of the Tobira instance configured here. If you want to be more restrictive, you can also always add content within Tobira itself.

Reply:
I took the liberty of adapting your suggestion a bit, but something to this effect is now added as well. :+1: